tests/plugins/test_tpu_spawn.py

```diff
@@ -114,7 +114,6 @@ def test_tensorboard_ddp_spawn_cleanup(use_list, tmpdir):
     assert tensorboard_logger._experiment is not None
```

Comment:
I think we should remove the test entirely. Its only purpose was to test `_clean_logger`